src/tool/abi/abi.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.13.35913.81 d17.13
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "abi", "abi.vcxproj", "{F025D704-CA01-4C93-9097-D0C7BF9CCA07}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
+		Release|x64 = Release|x64
+		Release|x86 = Release|x86
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{F025D704-CA01-4C93-9097-D0C7BF9CCA07}.Debug|x64.ActiveCfg = Debug|x64
+		{F025D704-CA01-4C93-9097-D0C7BF9CCA07}.Debug|x64.Build.0 = Debug|x64
+		{F025D704-CA01-4C93-9097-D0C7BF9CCA07}.Debug|x86.ActiveCfg = Debug|Win32
+		{F025D704-CA01-4C93-9097-D0C7BF9CCA07}.Debug|x86.Build.0 = Debug|Win32
+		{F025D704-CA01-4C93-9097-D0C7BF9CCA07}.Release|x64.ActiveCfg = Release|x64
+		{F025D704-CA01-4C93-9097-D0C7BF9CCA07}.Release|x64.Build.0 = Release|x64
+		{F025D704-CA01-4C93-9097-D0C7BF9CCA07}.Release|x86.ActiveCfg = Release|Win32
+		{F025D704-CA01-4C93-9097-D0C7BF9CCA07}.Release|x86.Build.0 = Release|Win32
```

Comment:
Do we need to think about arm64 or is that irrelevant?

Reply:
The build pipeline for the abi tool is specifically only compiling x86. I'm not aware of any infrastructure that is consuming this on arm64, so irrelevant for now. Adding it to the msbuild solution is easy - the bigger pain would be beefing up all the rest of the build and packaging pipelines infrastructure.